pom.xml

```diff
@@ -1302,6 +1302,23 @@ flexible messaging model and an intuitive client API.</description>
         <groupId>org.apache.rat</groupId>
         <artifactId>apache-rat-plugin</artifactId>
         <configuration>
+          <includes>
```

Comment:
we do we need to explicitly select files to include ?
isn't it better to let the plugin scan the whole directory ?
we could skip files without noticing it.

Reply:
I revert it from the #9184 assemble file. I removed it and only use exclude for this.